[2022-11-16 01:31:16,699] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_world.hello_task 2022-02-27T00:00:00+00:00 [queued]>
[2022-11-16 01:31:17,270] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: hello_world.hello_task 2022-02-27T00:00:00+00:00 [queued]>
[2022-11-16 01:31:17,303] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-11-16 01:31:17,329] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-11-16 01:31:17,361] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-11-16 01:31:17,962] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): hello_task> on 2022-02-27T00:00:00+00:00
[2022-11-16 01:31:18,063] {standard_task_runner.py:52} INFO - Started process 309 to run task
[2022-11-16 01:31:18,220] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'hello_world', 'hello_task', '2022-02-27T00:00:00+00:00', '--job-id', '57', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/index.py', '--cfg-path', '/tmp/tmp64cc3g4u', '--error-file', '/tmp/tmpxkr2xtnx']
[2022-11-16 01:31:18,271] {standard_task_runner.py:77} INFO - Job 57: Subtask hello_task
[2022-11-16 01:31:24,196] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_world.hello_task 2022-02-27T00:00:00+00:00 [running]> on host 001906ffd7c9
[2022-11-16 01:31:26,177] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=hello_world
AIRFLOW_CTX_TASK_ID=hello_task
AIRFLOW_CTX_EXECUTION_DATE=2022-02-27T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-27T00:00:00+00:00
[2022-11-16 01:31:26,452] {logging_mixin.py:104} INFO - hi everyone
[2022-11-16 01:31:26,479] {python.py:118} INFO - Done. Returned value was: hello world
[2022-11-16 01:31:29,837] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=hello_world, task_id=hello_task, execution_date=20220227T000000, start_date=20221116T013116, end_date=20221116T013129
[2022-11-16 01:31:33,256] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-11-16 01:31:33,453] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-11-16 01:31:33,846] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 309
[2022-11-16 01:31:34,163] {process_utils.py:66} INFO - Process psutil.Process(pid=309, status='terminated', exitcode=0, started='01:31:17') (309) terminated with exit code 0
